Services/Concurrency/ConcurrencyConfig.cs

```diff
@@ -207,6 +211,27 @@ public int MinDeviceTelemetryLoopDuration
             }
         }
 
+        /// <summary>
+        /// When sending telemetry for all the devices in a thread, slow down if the loop through
+        /// all the devices takes less than N msecs. This is also the minimum time between two
+        /// messages from the same device.
+        /// </summary>
+        public int MinDeviceReplayLoopDuration
+        {
+            get => this.minDeviceReplayLoopDuration;
+            set
+            {
+                if (value < 1 || value > MAX_LOOP_DURATION)
+                {
+                    throw new InvalidConfigurationException(
+                        "The min duration of the device telemetry loop is not valid. " +
+                        "Use a value within 1 and " + MAX_LOOP_DURATION);
```

Comment:
Use a value between 1 and " + MAX_LOOP_DURATION) 
or 
Use a value within the range of 1 and " + MAX_LOOP_DURATION) ...